test/unit/tasks/data_migration/fix_backend_config_path_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class Tasks::FixBackendConfigPathTest < ActiveSupport::TestCase
+  fixtures :backend_api_configs
+
+  setup do
+    @config_without_path = backend_api_configs(:empty_path)
+    @config_with_path    = backend_api_configs(:with_path)
+  end
+
+  test 'migrates all configs with empty path to have a backslash on it' do
+    assert_empty @config_without_path.path
```

Comment:
```suggestion
    assert @config_without_path.path.blank?
```
To fix [the oracle test](https://circleci.com/gh/3scale/porta/117572#tests/containers/1).